documentation/SDKs/javascript/README.md

```diff
@@ -7,352 +7,264 @@
 ### Installing the JavaScript SDK for Nether
 
 ## Usage
-
-### Analytics
-
-#### Active sessions - daily
-You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsDaily();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Day + '-' + sessions[0].Month + '-' + sessions[0].Year);
-```
-#### Active sessions - monthly
-You can use the ***Nether.Analytics.GetActiveSessionsMonthly*** method to get the total active sessions per month. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsMonthly();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Month + '-' + sessions[0].Year);
+To use the nether SDK include the nether script in your project. Reference the script in the webpage using 
+```html
+<script src="/scripts/nether.js"></script>
+```
+```javascript
+var config = {
+        netherClientId: '<client Id>',
+        netherClientSecret: '<client secret>',
+        facebookAppId: '<facebook app Id>',
+        netherBaseUrl: '<nether url>'
+    }
+    nether.init(config, facebookInitailised, netherInitialised);
```

Comment:
@stuartleeks I would recommend that we prioritize the work on the JavaScript SDK to make the optional authentication using a custom provided Nether. The rest of the changes are better to defer for a few more days and are probably much easier to accomplish as we start seeing the pattern how to use this SDK. If you agree, then respond here and @AndyCross can go ahead with the work during the time he has left helping out.